storage/app/releases.dist.php

```diff
@@ -113,6 +113,20 @@
                 72, // WASM-4
             ],
         ],
+        Emulators::PCSX2 => [
+            //'minimum_version' => '?',
+            //'latest_version' => '?',
+            'latest_version_url' => 'https://pcsx2.net/downloads',
+            'name' => 'PCSX2',
+            'handle' => 'PCSX2
+            'active' => true,
+            'integration_id' => Emulators::PCSX2,
+            'link' => 'https://pcsx2.net',
+            'description' => '<strong>NOTE:</strong> The PCSX2 standalone is the only emulator officially supporting achievements for PlayStation 2.',
+            'systems' => [
+                21, // PlayStation 2
+            ],
+        ],
```

Comment:
This isn't going to work. Assuming you did define the Emulators::PCSX2 enum, you're also missing a quote on line 121.

However, more importantly, this logic is for clients we maintain (see https://github.com/RetroAchievements/RAInterface/blob/master/RA_Emulators.h), where we can force the user to upgrade.

PCSX2 is using the non-emulatorID API to initialize the DLL, so the version validation code wouldn't be called anyway. Furthermore, when not using the DLL, no validation occurs.

You need to shoehorn something in akin to the RetroArch entry:
```
        101 => [
            'name' => 'PCSX2',
            'handle' => 'PCSX2',
            'active' => true,
            'link' => 'https://pcsx2.net/guides/basic-setup/',
            'description' => '<strong>NOTE:</strong> The PCSX2 standalone is the only emulator officially supporting achievements for PlayStation 2.
Download from <a href="https://pcsx2.net/downloads">pcsx2.net</a>.',
            'systems' => [
                21, // PlayStation 2
            ],
        ],
```

Reply:
Lastly, do we want this featured as the third item in the list (below RetroArch and RALibretro, and above everything else)? I think it's probably fine there.